kuma/static/styles/minimalist/components/subscriptions/_subscription-form.scss

```diff
@@ -29,6 +30,10 @@
                 margin: 0;
             }
 
+            label {
+                display: flex;
+            }
```

Comment:
I think this might cause more problems than it solves. With this set to `flex`, the alignment of the `checkbox` is a bit off in Chrome:

![Screenshot 2020-04-15 at 12 58 08](https://user-images.githubusercontent.com/10350960/79329973-fe437d80-7f18-11ea-837a-9fc50258ea9b.png)

The alignment in Firefox is also very different and feels odd being centered with the copy:

![Screenshot 2020-04-15 at 12 58 21](https://user-images.githubusercontent.com/10350960/79330058-26cb7780-7f19-11ea-87a2-2adb8ef0ab9d.png)

If we forego `flex` and set the `font-size` to `$tiny-font-size` for the label I feel we get a better end result that is consistent between browsers:

![Screenshot 2020-04-15 at 12 58 29](https://user-images.githubusercontent.com/10350960/79330131-49f62700-7f19-11ea-8471-790fe18fe617.png)

Reply:
It looks like it just needed an `align-items` set to make it look right in Firefox:
<img width="498" alt="ff" src="https://user-images.githubusercontent.com/650/79365588-cfc5a280-7f18-11ea-9f29-19c778b96717.png">

Reply:
That works!